pay-per-app-idea.md

```diff
@@ -6,3 +6,5 @@ Include your bitcoin public address in the comment section of your pull request.
 
 #### Coin operated file viewer 
 - * Pay some coin to access an encrypted file while not involving 3rd parties in the transaction.
+- App that records all your financial transactions using something like Plaid API
```

Comment:
Speaking to folks from MIT today who have a hardware solution for this that they are working on and could integrate Blockstack